Gem/Code/Source/Components/CharacterComponent.cpp

```diff
@@ -132,7 +132,22 @@ namespace MultiplayerSample
 
     void CharacterComponent::OnTranslationChangedEvent([[maybe_unused]] const AZ::Vector3& translation)
     {
-        OnSyncRewind();
+        if (m_physicsCharacter == nullptr)
```

Comment:
This is almost just a copy paste of the sync rewind code.. feels a bit messy like this.

Reply:
Agreed, SyncRewind seems to only run server side while OnTranslationChangedEvent can be client. This can probably be collapsed with appropriate ownership checks.